vendors/vendor/boards/board/CMakeLists.txt

```diff
@@ -201,18 +201,37 @@ afr_set_board_metadata(KEY_IMPORT_PROVISIONING "TRUE")
 # )
 
 # OTA
-# afr_mcu_port(ota)
-# target_sources(
-#     AFR::ota::mcu_port
-#     INTERFACE "${portable_dir}/ota/aws_ota_pal.c"
-# )
-# Choose which backend to enable. Link to AFR::ota_mqtt to enable MQTT, link to AFR:ota_http to enable HTTP.
-# target_link_libraries(
-#     AFR::ota::mcu_port
-#     INTERFACE
-#         AFR::ota_mqtt
-#         AFR::ota_http
-)
+#afr_mcu_port(ota)
+#target_sources(
+#    AFR::ota::mcu_port
+#    INTERFACE
+#        "${portable_dir}/ota_pal_for_aws/ota_pal.c"
+#        "${portable_dir}/ota_pal_for_aws/ota_pal.h"
+#        # Add platform specific files that are required to build ota_pal.c.
```

Comment:
```suggestion
#        # Add platform specific files and helpers that are required to build ota_pal.c.
```

Reply:
what are helpers in this context?

Reply:
Doesn't this fall under platform specific files? I'm just wondering if saying helpers in addition to this would be confusing to some.